src/main/java/seedu/dictionote/ui/MainWindow.java

```diff
@@ -391,6 +391,9 @@ private void handlePanelVisibility(UiActionOption uiActionOption, boolean visibl
         case DICTIONARY_CONTENT:
             setPanelVisibility(dictionaryContentDisplay, visible);
             break;
+        case DICTIONARY_DEFINITIONS:
+            setPanelVisibility(dictionaryContentDisplay, visible);
+            break;
```

Comment:
the UI here refers to the panel, not the model definition. When using ListDefinition, you use UiActionOption.DICTIONARY_LIST, this will open the list panel to show the definition

The UIActionOption refer to the Panel, not the model. hence DICTIONARY_DEFINITION do not exist as no such panel exist.

Reply:
Ok thanks! I've edited the code